dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/resource/entity/FileResource.java

```diff
@@ -170,7 +171,19 @@ public Response getExport(
         outputStream -> fileService.exportFiles(outputStream, query(filtersParam), type);
 
     // Make this similar to client-side export naming format
-    val fileName = String.format("repository_%s.%s", (new SimpleDateFormat("yyyy_MM_dd").format(new Date())), type);
+    SimpleDateFormat formatter = new SimpleDateFormat("MMM dd yyyy HH:mm:ss.SSS zzz");
+    long epochTime = 0L;
+    String currentTime = formatter.format(new Date());
+
+    try {
+      Date currentDate = formatter.parse(currentTime);
+      epochTime = currentDate.getTime();
+
+    } catch(ParseException e){
+      e.printStackTrace();
```

Comment:
We don't want to suppress the exception. We can do something like the following:

```java
...
  @SneakyThrows
  public Response getExport(...) {

   ...

    try {
      val currentDate = formatter.parse(currentTime);
      epochTime = currentDate.getTime();
    } catch(ParseException e){
      log.error(e.getMessage());
      throw e;
    }

    ...
```

We are logging the error message of the exception through `log.error()`  and then continuing to raise it. We have exception mappers in place such that uncaught exceptions get mapped to appropriate HTTP responses, in this case, it would map correctly to an HTTP Error 500. 

Also we can use the lombok annotation `@SneakyThrows` so that we can get away with not declaring checked exceptions, hence the sneakiness.